engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/desugar/Imports.scala

```diff
@@ -50,7 +50,8 @@ case object Imports extends IRPass {
             val parts = newName.parts
             if (parts.length == 2) {
               i.copy(
-                name = newName.copy(parts = parts :+ mainModuleName),
+                name =
+                  newName.copy(parts = parts :+ mainModuleName.duplicate()),
```

Comment:
There was a report:
```
 Duplicate found after org.enso.compiler.pass.desugar.Imports$: org.enso.compiler.core.ir.Name$Literal@56476c16:Main
   referenced by org.enso.compiler.core.ir.Name$Qualified@76e9f00b:Standard.Base.Main
   referenced by org.enso.compiler.core.ir.Name$Qualified@314b9e4b:Standard.Test.Main
```
and with bb4aff2531ffa973f3d988628334ae3a27cc4d62 it is gone.